articles/machine-learning/resource-known-issues.md

```diff
@@ -425,6 +425,16 @@ method, or from the Experiment tab view in Azure Machine Learning studio client
   2. Enter `pip freeze` and look for `tensorflow`, if found, the version listed should be < 1.13
   3. If the listed version is a not a supported version, `pip uninstall tensorflow` in the command shell and enter y for confirmation.
 
+## Model explanations
+
+* **Sparse data not supported**: The model explanation dashboard breaks/slows down substantially with a large amount of features, therefore we currently do not support sparse data format. Additionally, general memory issues will arise with large datasets and large number of features. 
+
+* **Forecasting models not supported with model explanations**: Interpretability, best model explanation, is not available for Auto ML forecasting experiments that recommend the following algorithms as the best model: ForecastTCN, Average, Naive, Seasonal Average, and Seasonal Naive.
+
+* **Local explanation for data index**: The explanation dashboard does not support relating local importance values to a row identifier from the original dataset if the dataset is greater than 5000 datapoints as the dashboard randomly downsamples the data. However, the dashboard shows raw dataset feature values for each datapoint passed into the dashboard under the Individual feature importance tab. Users can map local importances back to the original dataset through matching the raw dataset feature values.
```

Comment:
Also "Individual Feature Importance" is not supported for AutoML forecasting.

Reply:
Isn't that just assumed if Interpretability as a whole isn't supported for AutoML forecasting?

Reply:
No. forecasting has regression models and explanations are supported for those models. However, the "Individual feature importance" tab is just not supported for forecasting because of complexity in their data pipelines.

Reply:
Ah I see. Thanks for the clarification